patches/node/worker_thread_add_asar_support.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fedor Indutnyy <indutny@signal.org>
```

Comment:
```suggestion
From: Fedor Indutny <indutny@signal.org>
```
extra `y`? :)

Reply:
Well, it is a story for another day I guess.